welcome/install-d-locally.md

```diff
@@ -1,7 +1,7 @@
 # D lokal installieren
 
 Auf der [Website der Programmiersprache](https://dlang.org) wird stets die neuste
-Version des Referenzkompilers **DMD** (Digital Mars D) zum [Download](http://dlang.org/download.html)
+Version des Referenz-Compilers DMD (Digital Mars D) zum [Download](http://dlang.org/download.html)
```

Comment:
Die ganzen Änderungen in welcome hast du schon im anderen PR, bitte hier raus nehmen damit die separat behandelt werden können